tests/lib/schema_validation_test.py

```diff
@@ -162,3 +205,178 @@ def test_invalid_state(self, default_data):
         with pytest.raises(js.ValidationError) as err:
             libnmstate.validator.validate(default_data)
         assert 'bad-state' in err.value.args[0]
+
+
+class TestVlanFiltering(object):
+    def test_basic_configuration(self, default_data):
+        libnmstate.validator.validate(default_data)
+
+    def test_bridge_vlan_filtering_only_values_on_schema(
+            self, default_data, bridge_state
+    ):
+        invalid_vlan_filtering_state = self._get_bridge_vlan_filtering_state(
+            vlan_filtering_enabled=True, invalid_key='ka-boom!'
+        )
+        bridge_state[LB.CONFIG_SUBTREE][
+            LB.OPTIONS_SUBTREE
+        ] = invalid_vlan_filtering_state
+        with pytest.raises(js.ValidationError) as err:
+            libnmstate.validator.validate(default_data)
+        assert "('invalid_key' was unexpected)" in err.value.args[0]
+
+        valid_vlan_filtering_state = self._get_bridge_vlan_filtering_state(
+            vlan_filtering_enabled=False
+        )
+        bridge_state[LB.CONFIG_SUBTREE][
+            LB.OPTIONS_SUBTREE
+        ] = valid_vlan_filtering_state
+        libnmstate.validator.validate(default_data)
```

Comment:
This code is probably not needed, since it's covered in [test_trunk_port_native_vlan](https://github.com/nmstate/nmstate/pull/440/files#diff-d4e1478b9ed666021932b2c056397a85R305)